src/EventStore.Core/Services/Storage/ReaderIndex/IndexReader.cs

```diff
@@ -380,6 +381,16 @@ public class IndexReader<TStreamId> : IndexReader, IIndexReader<TStreamId> {
 					indexEntries = tableIndex.GetRange(streamId, mid, mid + maxCount - 1);
 					if (indexEntries.Count > 0) {
 						nextEventNumber = indexEntries[0].Version + 1;
+					} else {
```

Comment:
the comment above needs adjustment from `two` to `three`:
... `two` explicit continue cases
... the `two` continue cases always narrow

Reply:
(or just remove `two`)

Reply:
thanks, done!